mixins/localize-dynamic-mixin.js

```diff
@@ -0,0 +1,57 @@
+import { getLocalizeOverrideResources } from '../helpers/getLocalizeResources.js';
+import { LocalizeMixin } from './localize-mixin.js';
+
+const fallbackLang = 'en';
+const defaultExportName = 'default';
+
+function warnMissingFiles(failures, resolvedLang) {
+	const { path } = failures[0].err.message.match(/(?:https?:\/\/.*?\/)(?<path>.*)\/.*\./).groups;
+	const langs = failures.map(f => f.lang).join('","');
+	console.warn(
+		'WARNING: Unacceptable in production. Consider Manually Retrieved Resources method.\n\n' +
+		`Failed to load translation resources for languages "${langs}" from ${path}.`,
+		`Falling back to "${resolvedLang}". `);
+}
```

Comment:
Definitely open to dealing with this differently.

Reply:
I'd suggest removing the "Unacceptable in production" bit since chances are these types of problems _will_ sneak into production and it doesn't look great if clients see this.

Maybe just drop the first line entirely?